### DISCUSSION TITLE
Secrets logging disable

### DISCUSSION THREAD

**Discussion by marconimedeiros2** [dataplane-app/dataplane#370] [Ideas & feature requests]
Having the option to disable **Secret** on the (output) prompt could be nice, at least in the development environment.

Ignoring some server data, such as the file path, has been a bit of a hindrance to our development and deployment process.

**Comment by saul-data** [comment]
I think the database name dataplane is set as a secret and that comes up a lot with directory structure. I will move this to an issue and we can look to move that to a non-secret. It would be good to maybe have a call to understand your issue more precisely and see what we can do about that.

**Comment by saul-data** [comment]
@marconimedeiros2 this has been improved in the latest release: https://github.com/dataplane-app/dataplane/releases/tag/v0.2.0
